packages/flutter_tools/gradle/src/main/kotlin/flutter.gradle.kts

```diff
@@ -1,33 +1,6 @@
 // Copyright 2014 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import FlutterPluginKts
 
 apply<FlutterPluginKts>()
-
-class FlutterPluginKts : Plugin<Project> {
```

Comment:
Thank you for continually making this file smaller by extracting functionality into their own files.